README.md

```diff
@@ -1,2 +1,102 @@
-# task_hii_landuse
-Earth Engine task for calculating land use driver
+HII LAND USE DRIVER
+---------------
+
+## What does this task do?
+
+This task calculates the (unitless) "influence" of land use on the terrestrial surface as one of the key
+drivers for a combined [Human Influence Index](https://github.com/SpeciesConservationLandscapes/task_hii_weightedsum). "Influence" is a pressure score based on a combination of land use/land cover and population density.
+
+The source landcover data are from the [ESA CCI Landcover Dataset](http://www.esa-landcover-cci.org/). The population density cells are from the [Gridded Population of the World (GPW) dataset developed by the Centre for International Earth Science Information Network (CIESIN)](https://sedac.ciesin.columbia.edu/data/collection/gpw-v4). This dataset models the distribution of the global human population on a 5 year cadence beginning in 2000 at a spatial resolution of 30 arc-seconds (~1km).
+
+For any given task date between two available GPW input images, the estimated population density is linearly interpolated to the specific task date. If the task date either precedes or is succeeds the available GPW images, the first or last available GPW image are used respectively. These values are bilinearly interpolated to an ~300m x 300m grid.
+
+Land use classes are given unique weights representing the human pressure associated with each land use. Natural land cover classes or only given weights if the population density in a given cell is greater than 0.
```

Comment:
Maybe make reference to Venter et al., and/or include some other articulation of why the weightings we are using were chosen. This might be a good time to circle back to the cards regarding drivers, technical articulation, and documentation...